why-jsonschema/index.qmd

```diff
@@ -254,12 +258,35 @@ What decision was made, use the form "We decided on CHOICE because of
 REASONS."
 :::
 
-TBD
+We decided to use `jsonschema` for automating checks on metadata. It is
+a great match for our use case: it offers a single functionality with a
+robust implementation, integrates into our codebase with minimal work,
+and makes it easy to keep Sprout up-to-date with the Data Package
+standard.
+
+Although `frictionless-py` implements the Data Package standard
+directly, it is less mature and less widely used than `jsonschema`. We
+have found some inconsistencies in its verification/validation behaviour
+and feel that we would need to customise it using somewhat brittle and
+inelegant workarounds for it to fit into our workflow. Pydantic would
```

Comment:
```suggestion
inelegant workarounds for it to fit into our workflow. `Pydantic` would
```

Reply:
I tried to follow what the authors of the packages do with using these quotes. The Pydantic people call it Pydantic without quotes (making it into a normal proper name I guess), but the `jsonschema` people use quotes. So I think it's either Pydantic or `pydantic`, if that makes sense.